db/entity/DiscordServerConfig.ts

```diff
@@ -19,6 +28,6 @@ export class DiscordServer {
   @Column("jsonb", { nullable: false, default: {} })
   starkyModuleConfig: StarkyModuleConfig;
 
-  @OneToMany((type) => DiscordMember, (member) => member.discordServer)
+  @OneToMany((type) => DiscordMember, (member) => member.DiscordServerConfig)
   members: DiscordMember[];
 }
```

Comment:
this looks good, but would probably need not to REPLACE discordServer but be a new entity